src/Integration.Vsix/CFamily/VcxProject/CmdBuilder.cs

```diff
@@ -165,14 +164,22 @@ internal void AddOptFromProperties(IVCRulePropertyStorage properties)
             var structMemberAlignment = GetPotentiallyUnsupportedPropertyValue(properties, "StructMemberAlignment", "");
             AddCmdOpt(ConvertStructMemberAlignment(structMemberAlignment));
 
-            var compileAs = properties.GetEvaluatedPropertyValue("CompileAs");
-            AddCmdOpt(ConvertCompileAsAndGetLanguage(compileAs));
-
             // Additional options
             var additionalOptions = properties.GetEvaluatedPropertyValue("AdditionalOptions");
             AddCmdOpt(additionalOptions);
         }
 
+        private void AddLanguageFlags(IVCRulePropertyStorage properties)
+        {
+            var (compileAsFlag, languageStandardFlag) = languageFlagsProvider.GetLanguageConfiguration(
+                GetPotentiallyUnsupportedPropertyValue(properties, "CompileAs", ""),
```

Comment:
Please use string.Empty
```suggestion
                GetPotentiallyUnsupportedPropertyValue(properties, "CompileAs", string.Empty),
```